### PR TITLE
Configurable npc skills

### PR DESCRIPTION
**Pull request by world-onetop**
Add an option in the sandbox settings to allow setting the NPC skill level to 10.

 CN -- 在沙盒选项中添加可设置NPC技能等级为10的选项。

### CODE REVIEW

File: mods/SuperbSurvivorsContinued/media/lua/client/01_SuperSurvivorsOptions/SuperSuvivorsSandBoxOptions.lua
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/lua/client/01_SuperSurvivorsOptions/SuperSuvivorsSandBoxOptions.lua:1]: Careful, if ALL NPCs are set to skill 10, raiders will never miss and kill you.

File: mods/SuperbSurvivorsContinued/media/lua/client/00_SuperbSurviorModVariables/SuperSurvivorsSuitsList.lua
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/lua/client/00_SuperbSurviorModVariables/SuperSurvivorsSuitsList.lua:1]: Good catch, ALICEpack is actually an external mod.
非常好，ALICEpack 实际上是外部修改。

File: mods/SuperbSurvivorsContinued/media/lua/client/07_Spawning/SuperSurvivorSpawnCore.lua [hunk 1]
```diff
@@ -180,46 +180,55 @@ function SuperSurvivorSpawnWife(player)
...
     then
-        player:getModData().WifeID = 0;
+        -- set SuperSurvivor gender to girl
+        if WifeIsGirl == true then
+            gender = true
+        else
+            gender = (not player:isFemale())
+        end
+        --Control the number of  generated by wife. 
+        for i = 1, IsWifeSpawnNumb do
```
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/lua/client/07_Spawning/SuperSurvivorSpawnCore.lua:190]: How many NPCs do you plan to have? The problem with this mod is having a lot of loose files that breaks the game after a few hours.
你打算有多少个 NPC？这个模组的问题是有很多松散的文件，这些文件会在几个小时后破坏游戏。

 I personally don't care about this change, but other players on this mod might care.
我个人并不关心这个变化，但这个模组的其他玩家可能会关心。太多白痴不会读字 (笑)

File: mods/SuperbSurvivorsContinued/media/lua/shared/Translate/EN/Sandbox_EN.txt [hunk 2]
```diff
@@ -10,9 +10,18 @@ Sandbox_EN = {
 	Sandbox_SuperbSurvivorsContinued_Limit_Npcs_Spawn = "Limit NPC Spawns",
 	Sandbox_SuperbSurvivorsContinued_Limit_Npcs_Spawn_tooltip = "Maximum number of NPCs that can be active in-game. WIP - This is incomplete",
 
+	Sandbox_SuperbSurvivorsContinued_Perk_Level = "survivors skill Level is 10",
+	Sandbox_SuperbSurvivorsContinued_Perk_Level_tooltip = "Set the skill level of all survivors to 10.",
+
 	Sandbox_SuperbSurvivorsContinued_IsWifeSpawn = "Spawn Wife",
 	Sandbox_SuperbSurvivorsContinued_IsWifeSpawn_tooltip = "Spawns a wife follower at the start of the game in your group.",
 
+	Sandbox_SuperbSurvivorsContinued_IsWifeSpawnNumb = "Wife Numb",
```
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/lua/shared/Translate/EN/Sandbox_EN.txt:19]: Can you change "Wife Numb" to "Wife Count"?
可以改 "Wife Numb" 成 "Wife Count" 吗?

File: mods/SuperbSurvivorsContinued/media/lua/shared/Translate/EN/Sandbox_EN.txt [hunk 3]
```diff
@@ -10,9 +10,18 @@ Sandbox_EN = {
...
+	Sandbox_SuperbSurvivorsContinued_IsWifeSpawnNumb_tooltip = "Determine how many wife followers to spawn at the start of the game.",
+
+	Sandbox_SuperbSurvivorsContinued_WifeIsGirl = "Wife Is Girl"
```
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/lua/shared/Translate/EN/Sandbox_EN.txt:22]: Can you change "Wife Is Girl" to "Wife Is Female"?
可以改 "Wife Is Girl" 成 "Wife Is Female" 吗?

File: mods/SuperbSurvivorsContinued/media/sandbox-options.txt [hunk 4]
```diff
@@ -50,6 +50,17 @@ option SuperbSurvivorsContinued.IsWifeSpawn
 	translation = SuperbSurvivorsContinued_IsWifeSpawn,
 }
 
+option SuperbSurvivorsContinued.IsWifeSpawnNumb
+{
+	type = integer,
+	min = 1,
```
Comment (shadowhunter100) [inline mods/SuperbSurvivorsContinued/media/sandbox-options.txt:56]: May as well set the min to 0 for "no wife".
干脆把最小值设置为零吧, 有一些玩家不要助手... 😁

**Comment by world-onetop** [comment]
I will re-pull after I make the necessary changes.

**Comment by shadowhunter100** [comment]
> I will re-pull after I make the necessary changes.
That is fine, no need to hurry. I wasn't planning to continue work on this mod anyway. 
好啊, 不必急 - 事实上，我没有任何继续制作的计划

Your changes aren't little which is why I need to check it against introducing new issues.
你的改变并不小。这就是为什么我需要检查可能的新问题

**Pull request closed without merging**